tests/backup/backup_restore_basic_test.go

```diff
@@ -1348,7 +1347,7 @@ var _ = Describe("{ScheduleBackupUsingNamespaceLabel}", func() {
 	namespaceMapping := make(map[string]string)
 	bkpNamespaces = make([]string, 0)
 	JustBeforeEach(func() {
-		StartTorpedoTest("ScheduleBackupUsingNamespaceLabel", "Schedule backup and restore, using namespace label and resource label", nil, 58041)
+		StartTorpedoTest("BasicBackupAndScheduleBackupUsingNamespaceLabel", "Schedule backup and restore, using namespace label and resource label", nil, 58041)
```

Comment:
How about "Basic and Schedule backup and restore, using namespace label and resource label" just to be more descriptive?